condor-ap/condor_meter

```diff
@@ -782,6 +782,11 @@ def classadToJUR(classad):
             r.Network(val, storageUnit='b', phaseUnit=networkPhaseUnit, metric=attr, description=attr)
     r.Network(total_network, storageUnit='b', phaseUnit=networkPhaseUnit, metric="total")
 
+    if not setIfExists(r.ExecutePool, classad, 'LastRemotePool', "Pool Host"):
+        collector_host = get_collector_host()
+        if collector_host:
+            r.ExecutePool(collector_host, "Pool Host from COLLECTOR_HOST")
```

Comment:
Can we do some massaging here to handle

1. Multiple hosts
2. Sinful strings
3. Hosts with ports

I think this field should just be an XML list (as you suggsted) of FQDNS

Reply:
Sure thing!

Reply:
Ok I believe my latest push should handle all these cases, and include one or more `<ExecutePool>` tags in the XML